### ISSUE TITLE
PortableSettingsProvider.dll can't work on win11

### ISSUE THREAD

**Issue by cisko99za** [eibol/ffmpeg_batch#135]
PortableSettingsProvider.dll on my win 11 can't work. same blur window. and with override on high DPI from property exe file don't show correct text under the icons

**Comment by eibol** [comment]
I Guess than installer versión does not display this issue. I Will do some testing and let you know.

**Comment by cisko99za** [comment]
yes right. i talk about portable version. and ok, i can wait your test

**Comment by eibol** [comment]
Hi, after  many tests I've only been able to find out that it is a question of running portable version with admin rights in Windows 11
Neither portablesettings nor other settings seem to be related to this behaviour.

If application is not run as admin and it is portable, layout scaled for reasons I ignore.

**Comment by eibol** [comment]
Hi, after many tests playing with application settings and Windows compatibility settings, I've been unable to find out a definitive cause for this behaviour. I've disabled portablesettings dll but behaviour persists. It maybe related to specific user display settings.

[FFBatch_321_patched](https://drive.upm.es/s/ggWZxqlrXWV3ADH/download)

I've tested this release on a Windows 11 at 125 scale factor, and it looks good.
I may have to disable application auto-resizing for some some cases/users. 
Please let me know your results.

**Comment by cisko99za** [comment]
sorry for delay. your link is now not available ...

**Comment by eibol** [comment]
Hi, please use [this link](https://drive.upm.es/s/fEsvwtkgx90rNxb/download).

**Comment by cisko99za** [comment]
same. i'm on win 11 24H2 x64 and same blur window. see my screenshot on attach
![shot_01](https://github.com/user-attachments/assets/1869c2fd-adc1-4432-9bbc-b7ef82fa810e)

**Comment by eibol** [comment]
I see. I'm afraid that is the best quality, application was not designed taking into account scaling above 100%, so now it is not viable to redesign the layout. Dpi text autoscaling breaks layout, so it has to be disabled.

**Comment by eibol** [comment]
Thank. I know. 
Being a free application, this is probably the beginning of the end of its lifecycle, at least for many users.
Unless someone else has the time/resources for a redesignment.

**Comment by cisko99za** [comment]
don't worry, it's a good app. maybe you can suggest in home how to perform the reduction of the high dpi (maybe many users do n't know how to do it). and well also without dpi reduction it's a good tool
